### ISSUE TITLE
[Lab Submission] Small tasks

### ISSUE THREAD

**Issue by cpranav93** [carpentries-incubator/deep-learning-intro#431]
Points raised:

- [x] Link to sklearn docs: recommend "explained in the scikit-learn docs" instead of explained "here": solved in #427 
- [x] minor syntax issue (: instead of =) causing "missing alt text" on an imaage in ep 4 [link to source](https://github.com/carpentries-incubator/deep-learning-intro/blob/7f90b4abee39eb4ec3fbdb52a3b587dedbb20ba7/episodes/4-advanced-layer-types.Rmd#L552): solved in #428 
- [x] Change "open a terminal" to statement that supports all users (Windows/Mac/Linux)
